tools/validators/instance_validator/validate/handler.py

```diff
@@ -79,6 +80,15 @@ def _ValidateTelemetry(subscription: str, service_account: str,
   helper = TelemetryHelper(subscription, service_account)
   helper.Validate(entities, timeout)
 
+def _GetFilepathsFromDir(root_dir):
+  """ Takes in a directory and returns a filepath list for all YAML
+  files it finds. """
+  file_paths = []
+  for root, _, files in os.walk(root_dir, topdown=False):
+    for name in files:
+      if '.yaml' in name:
```

Comment:
Minor, but `name.endswith('.yaml')` is more specific to what we're checking for.